.github/workflows/update-engines-version.yml

```diff
@@ -0,0 +1,50 @@
+name: Update Engines Version
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: "Tag to check and update the engines version"
+        required: true
+
+jobs:
+  update_engines:
+    name: "Check and update @prisma/engines@${{ github.event.inputs.tag }}"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js
+        uses: actions/setup-node@v2
+
+      - uses: pnpm/action-setup@v2.0.1
+        with:
+          version: 5.15.1
+
+      - name: Check if tag is avaliable on npm
+        run: |
+          echo 'Checking @prisma/engines@${{ github.event.inputs.tag }}'
+          pnpx -p @prisma/ensure-npm-release enr update -p @prisma/engines -u ${{ github.event.inputs.tag }}
```

Comment:
`enr` is a bit cryptic, what is this?

Reply:
Its abbreviation for `@prisma/ensure-npm-release` cli

Reply:
https://github.com/prisma/ensure-npm-release